recipes/dataeval/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "dataeval" %}
+{% set version = "0.61.0" %}
+{% set sha256 = "bb79f19913a3e6f4732cbba08f9d71006c11cf917081b6b81649a65ee5c39e4f" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9, <3.12
+    - pip
+    - poetry-core
+    - poetry-dynamic-versioning
+
+  run:
+    - python >=3.9, <3.12
+    - cython <3
+    - hdbscan
+    - maite
+    - numpy
+    - pillow
+    - python-xxhash
+    - scipy
+    - scikit-learn
```

These are the dependencies according the package metadata. Maybe those are wrong but, if that is so, we need to open an issue upstream.

```suggestion
    - python >=3.9,<3.12
    - poetry-core >=1.0.0
    - poetry-dynamic-versioning
    - pip
  run:
    - python >=3.9,<3.12
    - hdbscan >=0.8.36
    - numpy >1.24.3
    - pillow >=10.3.0
    - scipy >=1.10
    - scikit-learn >=1.5.0
    - python-xxhash >=3.3
    - maite
  run_constrained:
    - matplotlib-base *
    - nvidia-cudnn-cu11 >=8.6.0.163
    - tensorflow >=2.14.1,<2.16
    - tensorflow-io-gcs-filesystem >=0.35.0,<0.37
    - tensorflow_probability >=0.22.1,<0.24
    - pytorch >=2.0.1,!=2.2.0
```